dominion/game.py

```diff
@@ -71,8 +71,9 @@ def call_reaction_effect(
         self.log(player.deck, f"[{event.name}] {args} {kwargs}")
         if not inspect.isabstract(reaction_events[event]):
             if player.choice(
-                f"Activate your {card.name} to the {card.name}?",
+                f"Activate your {card.name} to the {card.name}?",  # TODO: Make this make sense
```

Comment:
For example, "Activate your Moat in response to the Militia?"
But yes I agree this should be changed.

Reply:
"Activate your Moat to the Militia?" is very confusing.